libs/core/src/components/alert/alert.stories.ts

```diff
@@ -23,29 +25,42 @@ const meta: Meta<GdsAlert> = {
       control: 'select',
       options: ['alert', 'status'],
     },
+    timeout: {
+      control: 'number',
+      description:
+        'Time in milliseconds before the alert is automatically dismissed',
+    },
   },
+  args: {
+    variant: 'information',
+    role: 'alert',
+    buttonLabel: '',
+    innerHTML: html`<strong>Information</strong> Body text starts on the same
+      row as heading. A link (optional) always ends the message.`,
+  },
+  render: (args) => html`
```

Comment:
It's actually better to leave out the render function, since it breaks Storybooks ability to automatatically assign args. With a render function, everything needs to be manually forwarded and it's easy to forget to update it later.

Reply:
Updated this to use `argTablePropsFor` function and also no render except for variants story that has multiple `alert`